recourse_methods/mrmc_method.py

```diff
@@ -92,47 +92,85 @@ def alpha(dist: np.ndarray) -> np.ndarray:
 
 
 class MRM:
```

Comment:
I'd recommend following this class docstring example as described here: https://google.github.io/styleguide/pyguide.html#384-classes

(Move the class initialization details to just below the `class MRM:` definition and out of the `__init__()` method's docstring.)

Reply:
I fixed this in this file and added an [issue](https://github.com/jakeval/MRMC/issues/34) to review the rest of the repo.